src/Heading.tsx

```diff
@@ -7,10 +7,10 @@ export interface IHeading extends Omit<IText<HTMLHeadingElement>, 'as'> {
   as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 }
 
-export const Heading: React.FunctionComponent<IHeading> = props => {
+export const Heading = React.forwardRef<HTMLOrSVGElement, IHeading>((props, ref) => {
```

Comment:
```suggestion
export const Heading = React.forwardRef<HTMLHeadingElement, IHeading>((props, ref) => {
```

Reply:
We can't do this because of the flexible `as` prop right?

Reply:
Could work!